pyaerocom/vert_coords.py

```diff
@@ -112,14 +118,14 @@ def geopotentialheight2altitude(geopotential_height):
 
     Parameters
     ----------
-    geopotential_height
+    geopotential_height : float
```

Comment:
annotating the return of this function should add this info to the docs,
and it will provide type hints to your IDE
``` python
def geopotentialheight2altitude(geopotential_height: float) -> float:
```